src/app/users/users-achievements/users-achievements.component.ts

```diff
@@ -42,7 +42,6 @@ export class UsersAchievementsComponent implements OnInit {
         )
       });
     }, (error) => {
-      this.planetMessageService.showAlert('There was an error getting achievements');
```

Comment:
We should not remove the error message entirely.  If the error is for something other than the document does not exist we should show that.